specs/experimental/security-council-safe.md

```diff
@@ -23,7 +23,48 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Liveness checking Mechanism
+The Security Council uses a specially extended Safe multisig contract to provide additional security
+guarantees on top of those provided by the Safe contract.
+
+## Deputy guardian module
+
+The Security Council acts as the Guardian, which is authorized to activate the [Superchain Pause](../protocol/superchain-configuration.md#pausability) functionality and for [blacklisting](../experimental/fault-proof/stage-one/bond-incentives.md#authenticated-roles) dispute game contracts.
+
+However the Security Council cannot be expected to react quickly in an emergency situation.
+Therefore the Pausability Pass Through module enables the Security Council to delegate share this
+authorization with another account.
+
+The module has the following minimal interface:
+
+```solidity
+interface DeputyGuardianModule {
+   /// @dev The address of the Security Council Safe
+   function safe() external view returns(address);
+
+   /// @dev The address of the account which can pause superchain withdrawals by calling this module
+   function deputyGuardian() external view returns(address);
```

Comment:
So the SC will only need to give one account the ability to pause? What account is that?

Reply:
In practice it will be the Foundation, I'm never sure how much these specs should focus on describing the setup of OP Mainnet, or just the specific component being described.

Reply:
Yea that's fair, I'm not sure if we should mention it explicitly